src/docker/mod.rs

```diff
@@ -153,6 +153,23 @@ impl CruxClient {
             }
         })
     }
+
+    /// Function `entity-tx` requests endpoint `/entity-tx` via `POST` which retrieves the docs and tx infos
+    /// for the last document saved in CruxDB.
+    pub fn entity_tx(&self, id: String) -> Result<EntityTxResponse> {
+        let mut s = String::new();
+        s.push_str("{:eid ");
+        s.push_str(&id);
+        s.push_str("}");
+
+        let resp = self.client.post(&format!("{}/entity-tx", self.uri))
```

Comment:
I am actually really confused, shouldn't this be a `GET` request?

https://opencrux.com/docs#rest-entity-tx

Am I looking at the wrong route? 🤔

Reply:
It is a Post on docker. Also got confused, and this is why we will need a different module for peer server

Reply:
Wow, so wierd hahah